spdx-utils/src/main/kotlin/SpdxExpression.kt

```diff
@@ -116,12 +123,43 @@ sealed class SpdxExpression {
      */
     abstract fun validate(strictness: Strictness)
 
+    /**
+     * Return all valid license choices for this SPDX expression, by converting it to the
```

Comment:
nit: As `A AND A` would be a valid choice for `A OR B AND C` would it make sense to somehow mention that minimized expressions are regarded?

Reply:
I think I don't understand what you mean, this function does not minimize expressions, for example if the input expression is `(a AND a) OR (b AND b)` the output would contain `a AND a`  and `b AND b`.

Reply:
It's not worth the effort talking about.